lib/udx.js

```diff
@@ -16,6 +16,8 @@ module.exports = class UDX {
     this._buffer = null
     this._reallocMessage()
 
+    this._view64 = new BigUint64Array(this._handle.buffer, this._handle.byteOffset, this._handle.byteLength >> 3)
```

Comment:
can you move this before the reallocmessage call? so so the prop defs are before any calls